websites/Y/YouTube Music/dist/metadata.json

```diff
@@ -3,6 +3,12 @@
     "name": "Timeraa",
     "id": "223238938716798978"
   },
+  "contributors": [
+    {
+      "name": "Encrypted",
+      "id": "564434085708038144"
+    }
+  ],
```

Comment:
don't think you did 25% change

Reply:
Where is that even a rule? Didn't know lol